source/documentation/monitoring-an-app/application-metrics.md

```diff
@@ -1,107 +1,121 @@
-        matchLabels:
-          component: monitoring
-```
-You can view your current NetworkPolices with the following command:
+   kind: NetworkPolicy
+   apiVersion: networking.k8s.io/v1
+   metadata:
+     name: allow-prometheus-scraping
+     namespace: my-app-namespace
+   spec:
+     podSelector:
+       matchLabels:
+         app: rails-app
+     policyTypes:                                                                                                                                                         
+     - Ingress
+     ingress:
+     - from:
+       - namespaceSelector:
+           matchLabels:
+             component: monitoring
+   ```
+
+#### Querying metrics
+
+Now we've exposed our metrics and asked Prometheus to scrape application latency, we can head over to the UI and see our service appear in Prometheus [targets](https://prometheus.cloud-platform.service.justice.gov.uk/targets).
+
+Head to [Cloud Platform Prometheus](https://prometheus.cloud-platform.service.justice.gov.uk/graph) and use the following promql query to view the application latency (remembering to change the namespace value):
 
-```sh
-kubectl -n <namespace> get networkpolicies
+```
+http_server_request_duration_seconds_sum{namespace="my-namespace"}
 ```
 
+#### Example in full
+If you'd like to see the changes I've made to the [cloud-platform-multi-container-demo-app](https://github.com/ministryofjustice/cloud-platform-multi-container-demo-app), please see this [PR](https://github.com/ministryofjustice/cloud-platform-multi-container-demo-app/pull/7).
+
+
+#### More information on Service Monitors
+
+[CoreOS Blog on Prometheus Operator and ServiceMonitor](https://coreos.com/blog/the-prometheus-operator.html)
+[CoreOS README on Custom Resource Definitions](https://github.com/coreos/prometheus-operator#customresourcedefinitions)
+[Example ServiceMonitors](https://coreos.com/operators/prometheus/docs/latest/user-guides/running-exporters.html)
+
 #### Advisory note: Applications configured to use multiple processes
 
-If you're using a pre-forking web server (like unicorn or puma for Ruby, or gunicorn for Python) and have it configured to use multiple processes, then you need to use a Prometheus client library which supports exporting metrics from multiple processes. Not all the official clients do that. If you don't use a library which supports this, then requests to `/metrics` could be served by any of the processes, which would mean Prometheus sees inconsistent data on each scrape
+If you're using a pre-forking web server (like unicorn or puma for Ruby, or gunicorn for Python) and have it configured to use multiple processes, then you need to use a Prometheus client library which supports exporting metrics from multiple processes. Not all the official clients do that. If you don't use a library which supports this, then requests to `/metrics` could be served by any of the processes, which would mean Prometheus sees inconsistent data on each scrape.
```

Comment:
Almost everyone will be using puma, because that's the rails default. So, this section will apply to just about everyone. I'd recommend clarifying that prometheus-client is one of the libraries that supports multi-process metrics, and add a link to https://github.com/prometheus/client_ruby#aggregation-settings-for-multi-process-stores

Reply:
I found this difficult to do. I've added the statement about the ruby client with the link. Not sure about the how to show that it applies to everyone.